src/HealthCheck/HttpHealthCheck.php

```diff
@@ -0,0 +1,160 @@
+<?php
+declare(strict_types=1);
```

Comment:
Was this intentional?

Reply:
We did this for all the health checks. In theory we are not really violating anything with it, it's new stuff, that does require a higher version of php... We should probably migrate to a minimum version of 7.1 and switch versions to 0.3.